keras_cv/utils/preset_utils.py

```diff
@@ -138,11 +153,22 @@ def check_preset_class(
 ):
     """Validate a preset is being loaded on the correct class."""
     config_path = get_file(preset, config_file)
-    with open(config_path) as config_file:
-        config = json.load(config_file)
+    try:
+        with open(config_path) as config_file:
+            config = json.load(config_file)
+    except:
+        raise ValueError(
+            f"The specified preset  `{preset}` is unknown. "
+            "Please check documentation to ensure the correct preset "
+            "handle is being used."
+        )
     cls = keras.saving.get_registered_object(config["registered_name"])
-    if not isinstance(classes, (tuple, list)):
+    # Subclass checking
```

Comment:
This is where I was getting confused too. This first check isn't to actually check subclass, it is just making sure `classes` is an iterable.

Reply:
Yeah I was confused here too, reverted to isinstance here!

Reply:
maybe just delete the comment on this line, it doesn't really track (and duplicates below)

Reply:
Removed, thanks